sites/partners/src/components/listings/PaperListingForm/sections/ApplicationTypes.tsx

```diff
@@ -591,31 +610,31 @@ const ApplicationTypes = ({ listing, requiredFields }: ApplicationTypesProps) =>
         <Drawer.Content>
           <Card>
             <Card.Section>
```

Comment:
is this check needed? this was making The Select Language field not show after uploading the pdf. Why this was there in the first place?

https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/bloom-housing/bloom/5032
Acceptance Criteria
Select Language field will not disappear after uploading a file.